dagster/helm/dagster/values.yaml

```diff
@@ -98,7 +98,7 @@ dagster:
       - name: "k8s-example-user-code-1"
         image:
           repository: ghcr.io/pluralsh/containers/dagster/user-code-example
-          tag: 1.3.4-plural1.0.0
+          tag: 1.3.5-plural1.0.0
```

Comment:
are you sure this container has been built?  If not it might be worth keeping it at the prior version for now

Reply:
Fixed @michaeljguarino 
I checked, the image has been built

Reply:
kk